front/app/containers/Admin/projects/components/ProjectRow/ProjectMoreActionsMenu.test.tsx

```diff
@@ -111,28 +111,17 @@ describe('ProjectMoreActionsMenu', () => {
 
   describe('When user is a project moderator', () => {
     describe('project that user moderates', () => {
-      it('Has a button to copy but not delete a project', async () => {
+      it('Has no button to copy nor delete the project', async () => {
```

Comment:
The UI is correct, the folder moderator can copy a project that they moderator but reading this combination of wording for the test, It looks like they shouldn't have the copy option for the folder that they moderate. We could change the wording to reflect

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/12659000/220135140-76aff573-a962-4d62-83cd-d4c1a4955884.png">

Reply:
This test is about project moderators, not folder moderators. I'm not sure if I understand what you mean.

Reply:
From the wording, it looks like the project moderator should not see the copy option for the project that they moderate. Is that the intended direction?

Reply:
Yes, indeed.